### ISSUE TITLE
Importing cleave.js for react does not work in Version 1.3.1

### ISSUE THREAD

**Issue by mlaug** [nosir/cleave.js#330]
Using Version 1.3.1 and trying to import cleave for eact

`import Cleave from 'cleave.js/react';`

results in error

```
ERROR in ./src/xxx/yyy.js
Module not found: Error: Can't resolve 'cleave.js/react' in 'xx/yyy/zzz.js'
```

using Version 1.2.0 works fine.

**Comment by khaosdoctor** [comment]
I can confirm this is happening!

**Comment by mayurdange** [comment]
This seems to be because package.json contains
"version": "1.3.1",
   "files": [
    "src",
    "dist"
  ],

thus we only include folders src and dist

**Comment by khaosdoctor** [comment]
I don't know if this is really the problem, since the `dist` folder contains the `cleave-react.js` file too...

**Comment by mayurdange** [comment]
A quick and easy way to test, this is the case:
in any app using import Cleave from 'cleave.js/react';
with cleave 1.3.1 when we try to start/build the project, it gives an error
Module not found: Error: Cannot resolve module 'cleave.js/react' in

go to node_modules\cleave.js and put the file react.js (from version 1.2.1 or root of this repo)
try to start/build the project again, then it is OK.

I will try to provide a complete example soon.

**Comment by nosir** [comment]
Sorry guys,
I’ve merged the PR.
Will bump version and release when home.

**Comment by nosir** [comment]
Hi guys. Fix release in v1.3.2, sorry again.
Can I close this?

**Comment by khaosdoctor** [comment]
Things started working again :D
